apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -0,0 +1,67 @@
+@use '../fozzie-config' as f;
+
+$card-width: 328px;
+$card-image-height: 180px;
+$card-label-height: 72px;
+
+.c-card-wrapper {
+    container-type: inline-size;
+}
+
+.c-card-list {
+    margin: f.$spacing-f 0;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    gap: f.$spacing-e;
+    list-style: none;
+    padding: 0;
+}
+
+@container (max-width: 680px) {
```

Comment:
perhaps we could flip this to be mobile first, so center justified by default and then at a min of 680, we switch to the 'desktop' styling?